### PR TITLE
Cover Image & Genres Support

### PR DESCRIPTION
**Pull request by awb8593**
Cover images can now be added to the directory COMS-Console/data/cover_images with the naming convention [filename].jpg

Genres is also now an attribute for Games.

### CODE REVIEW

**Comment by zmb6893** [comment]
@awb8593 Looks good! Just add a little note in the Readme that tells people that when they are adding a game, they need to make sure to put the cover image in the directory that you have specified in the code!

File: backend/src/launcher.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 database_directory = "../data/games.json"
 games_directory = "/home/pi/ConsoleGames"
+cover_image_directory = "../data/cover_images"
```
Comment (zmb6893) [inline backend/src/launcher.py:8]: should let the user know that images will need to be stored in the ../data/cover_images

**Review by zmb6893 — CHANGES_REQUESTED** [review]
This time I remembered to submit the review

File: backend/src/launcher.py (hunk 1 above)
Comment (awb8593) [reply backend/src/launcher.py:8]: On it

**Review by m-dorsey — COMMENTED** [review]
Would it be possible to add a `times_played` (number) and `last_played` (date) attribute to the game data? This would help with the sorting and filtering methods we want to support. 

Both of these fields would be updated when the user clicks `Play` on a particular game

**Comment by awb8593** [comment]
@m-dorsey Yes, I will add those fields now. For the conventions for the dates, what makes it easiest for you on the front end? We can pretty much store it however we want in the backend

**Comment by awb8593** [comment]
@m-dorsey currently I'm storing last_played as a string so you should just be able to put in whatever convention works for you on the front end, let me know if this works for you or not

**Review by m-dorsey — APPROVED** [review]
👍

**Pull request merged**